#ifndef HAL_PIGPIO_OUTPUT
#define HAL_PIGPIO_OUTPUT

// Services and messages headers (generated)
#include "hal_pigpio_interfaces/srv/hal_pigpio_set_pwm_dutycycle.hpp"
#include "hal_pigpio_interfaces/srv/hal_pigpio_set_pwm_frequency.hpp"
#include "hal_pigpio_interfaces/srv/hal_pigpio_set_gpio_high.hpp"
#include "hal_pigpio_interfaces/srv/hal_pigpio_set_gpio_low.hpp"
#include "hal_pigpio_interfaces/srv/hal_pigpio_send_trigger_pulse.hpp"
#include "hal_pigpio_interfaces/srv/hal_pigpio_get_mode.hpp"

#endif
